client/components/theme-collection/index.tsx

```diff
@@ -101,13 +120,19 @@ export default function ThemeCollection( {
 					<div className="theme-collection__description">{ preventWidows( description ) }</div>
 				</div>
 				<div className="theme-collection__carousel-controls">
-					<Button className="theme-collection__see-all" onClick={ onSeeAll }>
+					<Button className="theme-collection__see-all" onClick={ onSeeAllAction }>
 						{ translate( 'See all' ) }
 					</Button>
-					<Button className="theme-collection__carousel-nav-button theme-collection__carousel-nav-button--previous">
+					<Button
+						onClick={ () => trackNavigationClick( 'left' ) }
```

Comment:
I think it's better to use prev/next rather than left/right. It's just slightly less ambiguous when considering RTL languages.

Reply:
Sure.